skyportal/plot.py

```diff
@@ -746,6 +749,11 @@ def photometry_plot(obj_id, user, width=600, height=300, device="browser"):
     plot.add_layout(LinearAxis(x_range_name="Days Ago", axis_label="Days Ago"), 'below')
 
     colors_labels = data[['color', 'label']].drop_duplicates()
+    height = (
+        500
+        if device == "browser"
+        else math.floor(width / aspect_ratio) + 18 * len(colors_labels) + 100
```

Comment:
Maybe add a comment to explain where the 18 comes from

Reply:
Will do!

Reply:
But where does the 18 come from?  Is that 1rem (16px) + 2?

Reply:
That would be my guess, but can't be certain as it's controlled by bokeh I think. This will all be a mute point and changed  once #1758 goes in and we stop using the `CheckboxWithLegendGroup`.